### ISSUE TITLE
Can you help me about error at compiling Ipopt under Cygwin/mingw

### ISSUE THREAD

**Issue by svigerske** [coin-or/Ipopt#133]
Issue created by migration from Trac.

Original creator: wendel

Original creation time: 2010-09-18 23:01:27

Assignee: ipopt-team

Version: 3.8

Hi people!

I would love to use Ipopt in my academic work. However, I have been some problems.

I am using Cygwin/Mingw 4.5. After any time trying to run configure and make scripts, I have the following message:

''make[3]: Entering directory `C:/cygwin/home/Wendel/novoIpopt-3.8.0/Ipopt/src/Com
mon'
echo "#ifndef HAVE_CONFIG_H" >bla
echo "#define HAVE_CONFIG_H" >>bla
echo "#endif" >> bla
cat ./IpoptConfig.h >> bla
/home/Wendel/novoIpopt-3.8.0/install-sh -c -m 644 bla /home/Wendel/novoIpopt-3.8
.0/include/coin/IpoptConfig.h
process_begin: CreateProcess(NULL, /home/Wendel/novoIpopt-3.8.0/install-sh -c -m
 644 bla /home/Wendel/novoIpopt-3.8.0/include/coin/IpoptConfig.h, ...) failed.
make (e=3): O sistema nÒo pode encontrar o caminho especificado.
make[3]: *** [install-exec-local] Error 3
make[3]: Leaving directory `C:/cygwin/home/Wendel/novoIpopt-3.8.0/Ipopt/src/Comm
on'
make4b93956e64: *** [install-am] Error 2
make4b93956e64: Leaving directory `C:/cygwin/home/Wendel/novoIpopt-3.8.0/Ipopt/src/Comm
on'
make6afa9fa918: *** [install-recursive] Error 1
make6afa9fa918: Leaving directory `C:/cygwin/home/Wendel/novoIpopt-3.8.0/Ipopt'
make: *** [install-recursive] Error 1
''

In other words, I get the error that the  systens cannot find the file: 

\Ipopt\src\contrib\LinearSolverLoader\.libs\libLinearSolverLoader.a

I have checked this file and it is in this folder. I do not Know what I can do for solve that. Does Anyone have any idea?

**Comment by svigerske** [comment]
Comment by @andrea5w created at 2010-09-22 22:34:26

Resolution: duplicate
